_posts/2016/2016-07-04-ep6-offsite.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Team epages 6 Offsite"
```

Comment:
http://web.archive.org/web/20130117225252/http://writersblock.ca/tips/monthtip/tipmar98.htm
I will change the wording, but I will keep the capitalization.